auctions/english-auction/packages/app/src/systems/Core/hooks/useWallet.ts

```diff
@@ -18,12 +18,12 @@ export const useWallet = () => {
   } = useQuery(
     ['wallet'],
     async () => {
-      const isConnected = await fuel.isConnected();
+      const isConnected = await fuel!.isConnected();
```

Comment:
it will throw an error if fuel is undefined.  This isn't always a great idea, but in this case we know fuel will be defined becaused of the `enabled` option we pass into `useQuery`.  Unfortunately, the ts compiler isn't able to pick up on this.  I think it's called the non-null assertion operator.